src/main.rs

```diff
@@ -200,20 +200,25 @@ fn main() {
 #[allow(clippy::items_after_statements, clippy::needless_raw_string_hashes)]
 fn print_opening(opts: &Opts) {
     debug!("Printing opening");
-    let s = format!(
-        "{}\n{}\n{}\n{}\n{}",
+    let s = concat!(
```

Comment:
I agree having this as a `static &str` makes sense but in that case, any reason not to just have this as a plain assignment? And skip the macros entirely?

i.e. something like:

```rust
r#".----. .-. .-. .----..---.  .----. .---.   .--.  .-. .-.
| {}  }| { } |{ {__ {_   _}{ {__  /  ___} / {} \ |  `| |
| .-. \| {_} |.-._} } | |  .-._} }\     }/  /\  \| |\  |
`-' `-'`-----'`----'  `-'  `----'  `---' `-'  `-'`-' `-'
The Modern Day Port Scanner."#;
```

Reply:
@PsypherPunk  Done :)